arch/arm64/configs/crocodile_defconfig

```diff
@@ -227,7 +227,7 @@ CONFIG_PLATFORM_MHU=y
 CONFIG_ARM_SMMU=y
 CONFIG_ARM_SMMU_V3=y
 CONFIG_REMOTEPROC=y
-CONFIG_IMX_REMOTEPROC=y
+CONFIG_IMX_REMOTEPROC=m
```

Comment:
We should try to keep it "builtin", same discussion as in https://github.com/hexagon-geo-surv/linux-leica/pull/118

Reply:
ACK - system architect responded, see the linked PR